src/main/config/checkstyle/checker.xml

```diff
@@ -95,7 +95,7 @@ limitations under the License.
     <module name="EqualsHashCode"/>
       <!-- Disallow unnecessary instantiation of Boolean, String -->
     <module name="IllegalInstantiation">
-      <property name="classes" value="java.lang.Boolean, java.lang.String"/>
```

Comment:
Why are we changing this ?

Reply:
Ok， I use a loop to convert byte[] to String instead of changing the checker.xml.